src/server-code/admin-server.js

```diff
@@ -36,6 +44,8 @@ export default function runAdminServer(serverFalcorModel) {
         <link rel="stylesheet" href="/pure-min.css">
         <link rel="stylesheet" type="text/css" href="/admin.css?h=${cssHash}">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="google-signin-client_id" content="${googleClientID}">
+        <script src="https://apis.google.com/js/platform.js" onload="GoogleAPILoaded=true" async defer></script>
```

Comment:
`GoogleAPILoaded` is a pretty good, unique name. Just to develop a standard and to remove all possible namespace clashes I'd love it if we did it like [this](https://github.com/thegazelle-ad/gazelle-server/blob/6d70a8f1c35573ad391d711727f363ae888984bc/__tests__/end-to-end/admin-tests/assets/checkServerRestarted.js#L1-L8) instead.

So maybe put the `window.THE_GAZELLE = {}` line inside a script tag in the `<head>` to make sure it's run before the google script (and put it before the Google script), and when you rebase onto my E2E tests you can remove that first line from the link I gave, and then make the global variable `window.THE_GAZELLE.googleAPILoaded`, sounds good?

Reply:
so just the `window.THE_GAZELLE = {};` line right?

Reply:
Yeah, just put that line in a different `<script>` tag that runs synchronously in the `<head>` tag and then do the `googleApiLoaded` as you're doing now, just inside that object defined above